apps/src/sites/studio/pages/pd/application/facilitator_application/new.js

```diff
@@ -7,6 +7,7 @@ document.addEventListener("DOMContentLoaded", function (event) {
   ReactDOM.render(
     <Facilitator1819Application
       {...getScriptData('props')}
+      saveProgressInSession={true}
```

Comment:
I don't know how I feel about this being a prop instead of something defined at the class level. We'll always want Facilitator1819Applications to save session progress - if that's the case then it doesn't seem like a prop to me - more like a class attribute.

Reply:
Yeah good point. I'll switch it to a static property